src/common/translations/en.json

```diff
@@ -174,9 +174,9 @@
         "keys": "Keys",
```

Comment:
@imstar15 Please examines those terms when you get time.

Keys should all be "wallet". We still have `"keys": "Keys"` definition left in messageIDs.

Tokens should all be named to "asset".